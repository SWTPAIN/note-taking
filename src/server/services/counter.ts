
let counter: number = 0
export function getCounter(): number {
    counter = counter + 1
    return counter
}
