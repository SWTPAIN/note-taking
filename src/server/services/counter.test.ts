import { getCounter } from './counter'

describe('Counter Test', () => {
  test('getCounter test', () => {
    const count = getCounter()
    expect(count).toMatchSnapshot()
  })
})
