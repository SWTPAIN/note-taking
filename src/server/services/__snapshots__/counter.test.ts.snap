// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Counter Test getCounter test 1`] = `1`;
